freqtrade/main.py

```diff
@@ -209,8 +211,18 @@ def create_trade(stake_amount: float) -> bool:
         raise DependencyException('No pair in whitelist')
 
     # Pick pair based on StochRSI buy signals
-    for _pair in whitelist:
-        if get_signal(_pair, SignalType.BUY):
+    with ThreadPoolExecutor() as pool:
+        awaitableSignals = []
+        for _pair in whitelist:
```

Comment:
I would use a list comprehension to build the awaitable signals list, it's more pythonic, tends to perform better (depends on python implementation) and gets rid of intermediate variables

```
awaitable_signals = [
    asyncio.wrap_future(pool.submit(get_signal, pair, SignalType.BUY))
    for pair in whitelist
]
```

Reply:
done.